framework/auth/core.py

```diff
@@ -437,6 +437,14 @@ def absolute_api_v2_url(self):
     def get_absolute_url(self):
         return self.absolute_api_v2_url
 
+    @property
+    def institutions_self_url(self):
```

Comment:
Minor: These properties are unnecessary--the `User` and `Node` classes would get pretty bloated if we had a two properties for each relationship.